module/utils.js

```diff
@@ -0,0 +1,9 @@
+export const LexArcanaUtils = {};
+
+/* -------------------------------------------- */
+/*                  Tools                       */
+/* -------------------------------------------- */
+LexArcanaUtils.ObjectToArray = function _ObjectToArray(obj)
+{
+	return Object.values(obj);
+}
```

Comment:
How about this simpler way to write it: 
```javascript
export const LexArcanaUtils = {
    /* -------------------------------------------- */
    /*                  Tools                       */
    /* -------------------------------------------- */
    function ObjectToArray(obj)
    {
	return Object.values(obj);
    }
}
```

Reply:
same, and its missing static keyword

Reply:
Hum after reflexion, it would be better with the class and static keywords, same goes for dice.js